client/my-sites/domains/domain-management/site-redirect/index.jsx

```diff
@@ -148,7 +149,19 @@ class SiteRedirect extends React.Component {
 								/>
 
 								<p className="site-redirect__explanation">
-									{ translate( 'All domains on this site will redirect here.' ) }
+									{ translate(
+										'All domains on this site will redirect here as long as this domain is set as your primary domain. ' +
+											'{{learnMoreLink}}Learn more{{/learnMoreLink}}',
+										{
+											components: {
+												learnMoreLink: (
+													<a
+														href={ localizeUrl( 'https://wordpress.com/support/site-redirect/' ) }
```

Comment:
Props on finding the `localizeUrl` method - I actually did not know it was a thing 😅 👏 
But I'd love it if you could slightly tweak the `<a>` component here, to be in line with other places we've included links to support. See for example here:

https://github.com/Automattic/wp-calypso/blob/180a2d16e0f92e4980c4e2da77c16851586e16e1/client/my-sites/domains/domain-management/transfer/transfer-out/transfer-prohibited.jsx#L24

So, basically:
 - `target="_blank" rel="noopener noreferrer"` ([just to be on the safe side](https://pointjupiter.com/what-noopener-noreferrer-nofollow-explained/))
 - and moving the URL itself to https://github.com/Automattic/wp-calypso/blob/180a2d16e0f92e4980c4e2da77c16851586e16e1/client/lib/url/support.js - where I see now that we _do_ use `localizeUrl`... and the old (but working, thanks to rewrite rules, of course) URL for the root of support docs 🤦 Feel free to update it too, but probably ideally we'd do it in a new PR (which you are welcome to do as well!). The reason for putting the URLs in this one file is that it's easier to search for them and when they need updating, it's also a much smaller PR. Case in point: https://github.com/Automattic/wp-calypso/pull/44210

Reply:
Thanks @klimeryk, all your suggestions make perfect sense.

Just one small comment: I  verified and the URL for the root of support was updated in master 3 months ago, so don't worry about it 👍